src/Home.jsx

```diff
@@ -23,13 +55,31 @@ class Home extends React.Component {
     this.setState({ toggledBar: !toggledBar });
   }
 
+  async callArea(selectedElement) {
+    const { sedimentaryList } = this.state;
+    try {
+      const response = await RestAPI.requestAreaSelected(selectedElement.name);
+      this.setState({
+        selectedArea: {
+          sedimentary_name: sedimentaryList.filter((item) => (item.code === response.sedimentary_code))[0].name,
+          ...response,
+        },
+      });
+    } catch (error) {
+      // TODO: Set state in a error (handling error)
+
+    }
+  }
+
   render() {
-    const { activeBlock, toggledBar } = this.state;
-    const { setActiveBlock } = this.props;
+    const {
+      selectedArea, selectedElement, toggledBar, data, sedimentaryList,
+    } = this.state;
+    const { setactiveArea } = this.props;
     const isToggled = toggledBar;
     return (
       <Layout
-        activeBlock={activeBlock}
+        activeArea={null}
```

Comment:
Why do you pass null instead of selectedArea

Reply:
Because in the Home component, I always want to hide header's behavior, like if there was always activeArea equals null. If I send anything, it will show header's style and that's not the behavior wanted.